src/psyclone/psyir/backend/sir.py

```diff
@@ -409,3 +418,52 @@ def unaryoperation_node(self, node):
         # There is an assumption here that the literal is a float (see #468)
         return ("{0}make_literal_access_expr(\"{1}{2}\", BuiltinType.Float)"
                 "".format(self._nindent, oper, result))
+
+    def ifblock_node(self, node):
+        '''This method is called when an IfBlock instance is found in
+        the PSyIR tree.
+
+        :param node: an IfBlock PSyIR node.
+        :type node: :py:class:`psyclone.psyGen.IfBlock`
+
+        :returns: the SIR code as a string.
```

Comment:
s/as a string// and is the SIR code Python?

Reply:
Changed to "SIR Python code."